cmd/build_test.go

```diff
@@ -127,27 +145,77 @@ func TestBuildCmd(t *testing.T) {
 		assert.Equal(t, "sd-artifacts", artifactsDir)
 	})
 
-	t.Run("Failed build cmd when too many args", func(t *testing.T) {
+	t.Run("Success build cmd with --meta", func(t *testing.T) {
 		root := newBuildCmd()
-		root.SetArgs([]string{"test", "main"})
+
+		root.SetArgs([]string{"test", "--meta", "{\"hoge\":\"fuga\"}"})
 		buf := bytes.NewBuffer(nil)
 		root.SetOut(buf)
+
+		expected := map[string]interface{}{
+			"hoge": "fuga",
+		}
+
+		launchNew = func(option launch.Option) launch.Launcher {
+			assert.Equal(t, expected, option.Meta)
+			return mockLaunch{}
+		}
+
 		err := root.Execute()
-		want := `Error: accepts 1 arg(s), received 2
-Usage:
-  build [job name] [flags]
+		want := ""
+		assert.Equal(t, want, buf.String())
+		assert.Nil(t, err)
+	})
 
-Flags:
-      --artifacts-dir string   Path to the host side directory which is mounted into $SD_ARTIFACTS_DIR. (default "sd-artifacts")
-  -e, --env stringToString     Set key and value relationship which is set as environment variables of Build Container. (<key>=<value>) (default [])
-      --env-file string        Path to config file of environment variables. '.env' format file can be used.
-  -h, --help                   help for build
-  -m, --memory string          Memory limit for build container, which take a positive integer, followed by a suffix of b, k, m, g.
-      --src-url string         Specify the source url to build.
-                               ex) git@github.com:<org>/<repo>.git[#<branch>]
-                                   https://github.com/<org>/<repo>.git[#<branch>]
+	t.Run("Success build cmd with --meta-file", func(t *testing.T) {
+		root := newBuildCmd()
 
-`
+		root.SetArgs([]string{"test", "--meta-file", "./testdata/test_meta.json"})
+		buf := bytes.NewBuffer(nil)
+		root.SetOut(buf)
+
+		expected := map[string]interface{}{
+			"hoge": "fuga",
+			"foo": map[string]interface{}{
+				"bar": "aaa",
+			},
+		}
+
+		launchNew = func(option launch.Option) launch.Launcher {
+			assert.Equal(t, expected, option.Meta)
+			return mockLaunch{}
+		}
+
+		err := root.Execute()
+		want := ""
+		assert.Equal(t, want, buf.String())
+		assert.Nil(t, err)
+	})
+
+	t.Run("Success build cmd with --meta and --meta-file", func(t *testing.T) {
+		root := newBuildCmd()
+
+		root.SetArgs([]string{"test", "--meta-file", "./testdata/test_meta.json", "--meta", "{\"hoge\":\"fuga\"}"})
+		buf := bytes.NewBuffer(nil)
+		root.SetOut(buf)
+
+		launchNew = func(option launch.Option) launch.Launcher {
+			return mockLaunch{}
+		}
+
+		err := root.Execute()
+		want := "Error: can't pass the both options `meta` and `meta-file`, please speify only one of them" + buildUsage
```

Comment:
```suggestion
		want := "Error: can't pass the both options `meta` and `meta-file`, please specify only one of them" + buildUsage
```
I think it's a `specify` typo